client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -683,11 +683,11 @@ export class WorkspaceView extends React.Component<any, any> {
       !isOracleWorkspace &&
       doesRootWorkspaceScratchpadContainRelevantContent;
 
-    const judgeRegex = /(?<=<judge>)(.*)(?=<judge>)/;
+    const judgeRegex = /<judge>(.*)<\/judge>/;
```

Comment:
Wait, do we want `<judge><judge>` or `<judge></judge>`?

When I first reviewed it was the former, but now it's the latter.

Reply:
The later - per Ben's request / reference to HTML